### ISSUE TITLE
[BITV] 9.1.4.11/2.2 - "Online status" Modal Dialog - The input field ("Clear status after") frame does not meet the contrast requirements for graphical user interface elements. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36968]
- [x] https://github.com/nextcloud/server/pull/36843
- [x] https://github.com/nextcloud/server/pull/36932

Page 2 — Dashboard
Dashboard - Nextcloud
https://try.nextcloud.com/ltd/a11y/index.php/apps/dashboard/

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-3dbfb7615a-v2-n2

</details>
